misc/python/materialize/cli/mzconduct.py

```diff
@@ -598,6 +601,19 @@ def run(self, comp: Composition, workflow: Workflow) -> None:
         print("Workflow has environment of", workflow.env)
 
 
+@Steps.register("sleep")
+class Sleep(WorkflowStep):
+    """Waits for the defined duration of time."""
+
+    def __init__(self, path: Path, duration: int) -> None:
+        super().__init__(path)
+        self._duration = duration
+
+    def run(self, comp: Composition, workflow: Workflow) -> None:
+        print(f"Sleeping {self._duration} seconds")
+        time.sleep(self._duration)
+
+
```

Comment:
I tried pretty hard to not include a sleep command because it's almost always a workaround for not having precise information somewhere else. It's fine if we really _need_ to add it, but I'd strongly prefer we fix everything else so that we can precisely measure "is okay" by instrospection. Keeping to that philosophy will really help us eventually delete all of mzcompose/mzconduct, I think.

Reply:
It's 100% a workaround for two things:

- Debezium startup just takes time (#4496)
- "waiting" some period of time to let materialize settle (#5012)

I've opened issues for each of these. Okay to address seperately?